diff --git a/arch/arm/mach-rockchip/rk3568/Kconfig b/arch/arm/mach-rockchip/rk3568/Kconfig
index baa51349..6a2b331e 100644
--- a/arch/arm/mach-rockchip/rk3568/Kconfig
+++ b/arch/arm/mach-rockchip/rk3568/Kconfig
@@ -27,6 +27,13 @@ config TARGET_QUARTZ64_RK3566
 	help
 	  Pine64 Quartz64 single board computer with a RK3566 SoC.
 
+config TARGET_TPI_LAZARUS_RK3568
+	bool "TPI LAZARUS/UVON"
+	select BOARD_LATE_INIT
+	help
+	  TPI LAZARUS/UVON industrial carrier with a Forlinx FET3568-C
+	  SoM (Rockchip RK3568), native SATA and UEFI boot.
+
 endchoice
 
 config ROCKCHIP_BOOT_MODE_REG
@@ -45,5 +52,6 @@ source "board/rockchip/evb_rk3568/Kconfig"
 source "board/anbernic/rgxx3_rk3566/Kconfig"
 source "board/hardkernel/odroid_m1/Kconfig"
 source "board/pine64/quartz64_rk3566/Kconfig"
+source "board/tpi/lazarus/Kconfig"
 
 endif
